src/noshow/dashboard/helper.py

```diff
@@ -50,7 +95,9 @@ def render_patient_info(
         )
     else:
         if current_patient:
-            if current_response.call_status != "Wordt gebeld":
+            if current_response.call_status == "Wordt gebeld":
+                st.warning("Deze patient wordt momenteel gebeld!", icon="📞")
```

Comment:
Is it possible to add which user is calling?

Reply:
Not at the moment, the user is only saved after saving the call response. I'll add this as an issue.